src/main/java/gtPlusPlus/xmod/forestry/bees/custom/GTPPBeeDefinition.java

```diff
@@ -567,22 +565,21 @@ public final ItemStack getMemberStack(EnumBeeType beeType) {
         return BeeManager.beeRoot.getMemberStack(bee, beeType.ordinal());
     }
 
-    public final IBeeDefinition getRainResist() {
-        return new BeeVariation.RainResist(this);
-    }
-
     private static ItemStack getSlagComb() {
-        return issStackValid(ItemUtils.getSimpleStack(GTPPBees.Comb_Slag, 1));
+        if (GTPPBees.Comb_Slag != null) {
+            ItemStack slagComb = GTPPBees.Comb_Slag.copy();
+            slagComb.stackSize = 1;
+            return slagComb;
+        }
+        return null;
```

Comment:
```suggestion
        return GTUtility.copyAmount(1, GTPPBees.Comb_Slag);
```
This will handle the null check on the stack already (import may be needed)